lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -45,5 +46,47 @@ __PACKAGE__->has_many(properties => 'OpenQA::Schema::Result::WorkerProperties',
 # TODO
 # INSERT INTO workers (id, t_created) VALUES(0, datetime('now'));
 
+sub seen(;$) {
```

Comment:
found it - it wants us to remove the prototype because they have no influence. I consider it documentation still